homeassistant/components/bring/todo.py

```diff
@@ -0,0 +1,157 @@
+"""Todo platform for the Bring! integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import TYPE_CHECKING
+
+from python_bring_api.exceptions import BringRequestException
+
+from homeassistant import config_entries, core
+from homeassistant.components.todo import (
+    TodoItem,
+    TodoItemStatus,
+    TodoListEntity,
+    TodoListEntityFeature,
+)
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import BringData, BringDataUpdateCoordinator
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(seconds=90)
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant,
+    config_entry: config_entries.ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the sensor from a config entry created in the integrations UI."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+
+    unique_id = config_entry.unique_id
+
+    if TYPE_CHECKING:
+        assert unique_id
+
+    for list_ in coordinator.data:
+        async_add_entities(
+            [
+                BringTodoListEntity(
+                    coordinator,
+                    list_=list_,
+                    unique_id=f"{unique_id} {list_['listUuid']}",
```

Comment:
How unique is the list uuid? If possible we still want to use the same unique_id for that. (So if someone re setups the integration not every reference to entities is lost)

Reply:
I want to able to setup the same list with 2 different users. Therefore the combination of the users unique id and the lists unique id.

Reply:
But why?

Reply:
That change was on my request, so maybe i can explain. I was helping @miaucl testing the component. I registered 2 accounts in HA, the first account had several Shopping Lists, they were all created correctly as Todo Lists in HA. The second account had only 1 shopping list, but it was a list that was shared with the first account, so no todo entity showed up under this account in HA. Without the knowledge about the listUUID being unique across different accounts, i wouldn't have expected this behaviour, hence considered this beeing a bug. 

Also, depending on which user added items to a list, Bring sends out push notifications (on the Bring App) that a list was changed, to all other members of a shared list, but not to the user who changed the list. This may be important, when setting up automations.

Reply:
Can we get some kind of user id?

Reply:
When creating the config entry, the uuid of the user is already set as the unique id, so yes, this is the user id

Reply:
oooooh right, for some reason I tought we picked the entry id, never mind then. this makes sense. Thanks